.gitignore

```diff
@@ -22,3 +22,12 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+node_modules
+radata
+
+.prettierrc
+test.ipynb
+.idea
+.DS_Store
+*.json
```

Comment:
would this ignore package.json...?

Reply:
Fixed. Igone only *.json from Examples folder